sound/talkie.h

```diff
@@ -1,6 +1,13 @@
 #ifndef SOUND_TALKIE_H
 #define SOUND_TALKIE_H
 
+/* DISABLE_TALKIE define will save memory by replacing the default
```

Comment:
Added comment to talkie.h for define, let me know if you would prefer somewhere else.